tests/qadence/test_quantum_model.py

```diff
@@ -44,7 +44,9 @@ def test_quantum_model_parameters(parametric_circuit: QuantumCircuit) -> None:
     assert len([i for i in model_psr.parameters()]) == 4
     assert len([i for i in model_ad.parameters()]) == 4
     embedded_params_psr = model_psr.embedding_fn(model_psr._params, {"x": torch.rand(1)})
+    embedded_params_psr.pop("orig_param_values", {})
```

Comment:
can you add a note here why we do this and a TODO to remove it again

Reply:
Is this still needed with the new pyqtorch version? I thought that fixed this?